client/src/tables/programFtes.js

```diff
@@ -277,11 +277,7 @@ Statistics.create_and_register({
 
     add("fte_prg_num", first_year_prg_num);
 
-    const min_planning_yr =
-      "{{planning_year_" +
-      _.min(_.map(extended_planning_years, (XX) => Number(XX.match(/\d+/)))) +
-      "}}";
-
+    const min_planning_yr = extended_planning_years[0];
```

Comment:
extended_planning_years is an array where order is guaranteed, so this should be safe